Client/Frontend/Widgets/OneLineCellAndFooter.swift

```diff
@@ -17,6 +17,18 @@ enum OneLineTableViewCustomization {
 
 class OneLineTableViewCell: UITableViewCell, NotificationThemeable {
     // Tableview cell items
+    
+    override var indentationLevel: Int {
+        didSet {
+            containerView.snp.remakeConstraints { make in
+                make.height.equalTo(44)
```

Comment:
Could we put the 44 in a constant with a relevant name in OneLineCellUX? We can also use it on line 74 down here.